parachain/scripts/helpers/weight-constants-template.hbs

```diff
@@ -0,0 +1,17 @@
+use frame_support::{
```

Comment:
Can we add a comment on top of this template that tells later viewers of the file how it is generated? 

Something like: 

// This file is auto generated through ..., refer to README.md#benchmark for details how to generate it